examples/flexbox/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014-2015, S. Chris Colbert
```

Comment:
Actually, the license should be moved to the root directory to cover everything.